server/datastore/mysql/software_installers_test.go

```diff
@@ -2,11 +2,14 @@ package mysql
 
 import (
 	"context"
+	"database/sql"
 	"testing"
+	"time"
 
-	"github.com/fleetdm/fleet/v4/server/fleet"
-	"github.com/fleetdm/fleet/v4/server/ptr"
-	"github.com/google/uuid"
+	"github.com/fleetdm/fleet/v4/server/contexts/ctxerr"
+	"github.com/fleetdm/fleet/v4/server/test"
+ 	"github.com/google/uuid"
+	"github.com/jmoiron/sqlx"
```

Comment:
seems like some imports were deleted by mistake here (given the compile errors/annotations)

Reply:
Oh yeah it looks like some things got messed up during the merge with main, I'll fix that asap